README.md

```diff
@@ -28,7 +28,13 @@ npm install nhsuk-prototype-kit
 Then, within your `app.js` file, add this line to the top:
 
 ```js
-const NHSPrototypeKit = require('nhsuk-prototype-kit')
+import NHSPrototypeKit from 'nhsuk-prototype-kit'
+```
+
+Or, if using CommonJS:
+
+```js
+const NHSPrototypeKit = await import('nhsuk-prototype-kit').then(m => m.default)
```

Comment:
Top-level await isn't supported in CommonJS

Thankfully [Node.js v20.19.0+](https://github.com/nodejs/node/releases/tag/v20.19.0) supports require with synchronous ESM

```suggestion
const NHSPrototypeKit = require('nhsuk-prototype-kit')
```